### PR TITLE
TSK-1177 update Planned of Task if only Due() was changed

### PR DESCRIPTION
**Pull request by BerndBreier**
Sonar is green: https://sonarcloud.io/dashboard?branch=TSK-1177&id=BerndBreier_taskana

### CODE REVIEW

File: lib/taskana-core/src/test/java/acceptance/task/UpdateTaskAccTest.java [hunk 1]
```diff
@@ -42,7 +43,35 @@
       userName = "user_1_1",
       groupNames = {"group_1"})
   @Test
-  void testUpdatePrimaryObjectReferenceOfTask()
+  public void should_updatePlanned_when_onlyDueWasChanged() throws Exception {
```
Comment (holgerhagen) [inline lib/taskana-core/src/test/java/acceptance/task/UpdateTaskAccTest.java:46]: These two tests would be better placed in ServiceLevelPriorityAccTest, don't you think?

File: lib/taskana-core/src/test/java/acceptance/task/ServiceLevelPriorityAccTest.java [hunk 2]
```diff
@@ -297,22 +297,25 @@ void testSetPlannedPropertyOnSingleTaskWithTaskUpdate()
       userName = "admin",
       groupNames = {"group_2"})
   @Test
-  void testSetDuePropertyOnSingleTask()
+  void should_SetDueOrPlannedProperty_when_TaskUpdateIsCalled()
```
Comment (holgerhagen) [inline lib/taskana-core/src/test/java/acceptance/task/ServiceLevelPriorityAccTest.java:300]: Would it be too much effort to separate the cases into separate tests? I would prefer that.

File: lib/taskana-core/src/test/java/acceptance/task/UpdateTaskAccTest.java (hunk 1 above)
Comment (BerndBreier) [reply lib/taskana-core/src/test/java/acceptance/task/UpdateTaskAccTest.java]: Done

File: lib/taskana-core/src/test/java/acceptance/task/ServiceLevelPriorityAccTest.java (hunk 2 above)
Comment (BerndBreier) [reply lib/taskana-core/src/test/java/acceptance/task/ServiceLevelPriorityAccTest.java]: Done

Comment (BerndBreier) [reply lib/taskana-core/src/test/java/acceptance/task/ServiceLevelPriorityAccTest.java]: Done

**Pull request merged**